### PR TITLE
[update title and doc]

### PR DESCRIPTION
**Pull request by thisis-Solomon**

### CODE REVIEW

**Comment by wilfredmulenga** [comment]
Hey Solomon. Thanks for the PR. I am not using `node-sass` for the project so I cant add it to readme. Please take a look at the issues created, if you can

File: public/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Project Zed</title>
+    <title>Zed Projects</title>
```
Comment (wilfredmulenga) [inline public/index.html:23]: going to keep the title as `Project Zed` for now

File: src/App.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import firebase from './config/firebase'
 import Card from './components/Card'
 import Navbar from './components/Navbar'
-import SignInModal from './components/Modals/SignInModal'
+import SignInModal from './components/Modals/SigninModal'
```
Comment (wilfredmulenga) [inline src/App.jsx:5]: lets keep this as `SignInModal` for now please

File: package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "react-dom": "^16.6.3",
     "react-modal": "^3.9.1",
     "react-redux": "^7.1.0",
-    "react-scripts": "^2.0.0",
+    "react-scripts": "^3.2.0",
```
Comment (wilfredmulenga) [inline package.json:19]: thanks. would definitely want to update this

**Review by wilfredmulenga — COMMENTED** [review]
added a few comments. hope you can have a look. Plus can you point this to the dev branch and not master

**Comment by thisis-Solomon** [comment]
1. When i tried to run it after cloning the project it demanded to install
node-sass...

2. SignInModal its passing an error, until you change it to SigninModal,
check the SignInModal import and and SignInModal.js they differ at the
middle the file name as 'in' and the imported as 'In'

**Comment by wilfredmulenga** [comment]
Cool, node-sass could be a dependency for a module. The typo on `SigninModal` could have been resolved in the dev branch. My kind suggestions:

Can you try pull from the dev branch and not master and make a PR from there. And if you can make PRs separate. e.g one PR for typo, another PR for version upgrades. Thanks

**Comment by wilfredmulenga** [comment]
will close this PR as it hasnt been updated for some time

**Pull request closed without merging**